src/web/clients/download/partial.md

```diff
@@ -2,28 +2,30 @@
 
 [![reqwest-badge]][reqwest] [![cat-net-badge]][cat-net]
 
-Uses [`reqwest::Client::head`] to get the content-length and validates the
-server setting the header [`reqwest::header::ContentRange`].
+Uses [`reqwest::Client::head`] to get the [Content-Length] of the response.
 
-If supported, downloads the content using [`reqwest::get`] by setting the
-[`reqwest::header::Range`] to do partial downloads printing basic progress
-messages in chunks of 10240 bytes.
+If supported, downloads the content using [`reqwest::Client::get`] and

```

Comment:
It took a few re-reads for me to get what was 'setting the Range header'.  Can we go with: 
If supported, downloads the content using reqwest::Client::get with the Range header set to do partial ...

Reply:
To be fair, the old phrasing wasn't much better :-).

Does this sound all right?

> The code then uses [`reqwest::Client::get`] to download the content in
chunks of 10240 bytes, while printing progress messages. The requested
ranges are specified using with the [Range] header.